openwisp_controller/config/apps.py

```diff
@@ -102,6 +104,21 @@ def connect_signals(self):
             sender=self.cert_model,
             dispatch_uid='cert_update_invalidate_checksum_cache',
         )
+        group_templates_changed.connect(
```

Comment:
When I was writing unit test for `group_templates_changed` yesterday, I found that it is being called on device group save. So I added a patch (not pushed) to do not send this signal on creation. Their is also a `post_save` signal for device group. So I want to ask whether to use `group_templates_changed` for both (creating and updating) or not?

Reply:
for consistency we can do it as in the other signals and not send it on create

Reply:
Ok :+1: 
I have another question:
why to call templates_changed handler on device group creation because there will be no devices associated with that group at the time of creation?

Reply:
then we can make that consistent too